config/activator.yaml

```diff
@@ -98,20 +98,8 @@ spec:
             value: config-observability
           - name: METRICS_DOMAIN
             value: knative.dev/serving
-        volumeMounts:
-        - name: config-logging
-          mountPath: /etc/config-logging
-        - name: config-observability
-          mountPath: /etc/config-observability
         securityContext:
           allowPrivilegeEscalation: false
-      volumes:
-        - name: config-logging
-          configMap:
-            name: config-logging
-        - name: config-observability
```

Comment:
Hmm, are we sure that the metric exporter logic doesn't read this on-start? 🤔

Reply:
I guess then the controller and other stuff should have it?

Reply:
I did some greps regarding that, yes. But better safe than sorry, let‘s ask those who know for sure.

Reply:
Looks good, the metrics exporter relies on the ConfigMap, see https://github.com/knative/pkg/blob/master/injection/sharedmain/main.go#L134